FreeRTOS/FreeRTOS-Kernel-Community-Supported-Ports
# Adding support for ZynqMPSoc(A53*4)

Description
-----------
Adding support for ZynqMPSoc(A53*4)


Test Steps
-----------
Tested with https://github.com/FreeRTOS/FreeRTOS-SMP-Demos/tree/main/FreeRTOS/Demo/CORTEX_A53_64-bit_UltraScale_MPSoC/RTOSDemo_A53/src


Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
